docs/uml/main/em/PrioritizedFlexStrat.puml

```diff
@@ -0,0 +1,59 @@
+@startuml
+'https://plantuml.com/activity-diagram-beta
+
+start
+
+if () then (excess feed-in)
+  if (EVCS exists?) then (yes)
+    :asset = EVCS;
+    (A)
+  endif
+  if (Battery exists?) then (yes)
+    :asset = Battery;
+    (A)
```

Comment:
Shouldn't this be `flex = Battery`? Because in the `excess load` path the `asset` was replaced by `flex`.